docs/Developers/Writing_Native_Bears.rst

```diff
@@ -597,5 +597,7 @@ For example, let's make an aspect bear named SpellingCheckBear.
                         message='Wrong spelling in word `{}`'.format(word),
                         aspect=DictionarySpelling('py'),
                     )
+.. note::
+    Coala uses 1-based line & column convention, i.e. the first line and first column are 1. However, some linter use 0-based convention. For example ``pylint`` uses 1-based line convention and 0-based column convention. We need to map these conventions, please refer to `Normalize line or Column convention <https://coala-api.netlify.com/developers/writing_linter_bears#normalize-line-or-column-numbers>`__.
```

Comment:
Why let people refer to linter bear doc? `@linter` is only for linter bears, and that does not help developers handle the situation in native bears.
Btw, you should use relative link when you refer to some other doc.

Reply:
Updated.